rules/rules-reviewed/eap8/eap6/eap8-resteasy.windup.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="eap8-resteasy"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of Resteasy for migration from version 3.15 to 4.x
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="eap" versionRange="[6,7]" />
+        <targetTechnology id="eap" versionRange="[8,)" />
+        <tag>eap8</tag>
+    </metadata>
+    <rules>
+        <rule id="eap8-resteasy-00001">
+            <when>
+                <or>
+                    <filecontent pattern="org.jboss.resteasy.resteasy-jettison-provider" filename="jboss-deployment-structure.xml" />
+                    <filecontent pattern="org.jboss.resteasy.resteasy-jettison-provider" filename="MANIFEST.MF" />
+                </or>    
+            </when>
+            <perform>
+                <hint title="The org.jboss.resteasy.resteasy-jettison-provider module has been removed" effort="5" category-id="mandatory">
+                    <message>The `org.jboss.resteasy.resteasy-jettison-provider` module has been removed. There is no documented solution for this issue.</message>
+                    <tag>eap8</tag>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="eap8-resteasy-00002">
+            <when>
+                <project>
+                    <artifact groupId="org.jboss.resteasy" artifactId="resteasy-jettison-provider"/>
+                </project>
+            </when>
+            <perform>
+                <hint title="The org.jboss.resteasy resteasy-jettison-provider artifact has been removed" effort="5" category-id="mandatory">
+                    <message>The `org.jboss.resteasy` `resteasy-jettison-provider` artifact has been removed. There is no documented solution for this issue.</message>
```

Comment:
```suggestion
                    <message>The `org.jboss.resteasy:resteasy-jettison-provider` artifact has been removed. There is no documented solution for this issue.</message>
```

Reply:
That will require a change to the test as well